MYshort.bib

```diff
@@ -314,12 +319,10 @@ @String{Piscataway
 	@String{Miami = "Miami, FL, USA"}
 	@String{Montreal = "Montreal, Quebec, Canada"}
 	@String{LosAlamitos = "Los Alamitos, CA, USA"}
-	@String{GI = "Gesellschaft f{\"u}r Informatik"}
-	@String{Bonn = "Bonn, Germany"}
-	@String{Singapore = "Singapore, Singapore"}
 
 @String{INFORMS = "INFORMS"}
  	@String{Catonsville = "Catonsville, USA"}
+ 	@String{Linthium = "Institute for Operations Research and the Management Sciences (INFORMS), Linthicum, MD, USA"}
```

Comment:
Would it make sense to abbreviate this in MYshort as "INFORMS, Linthicum, MD, USA"?

Reply:
I copied the address from an entry of 2004, and just shortended Maryland -> MD, would happily drop the non-address part, too.